src/framework-ondata.js

```diff
@@ -52,17 +52,19 @@ module.exports = function(payload, socketIndex) {
          if ( ! _.isFunction( this.nodeInstance.wrapper.fRunUpdater ) ) { 
 
              // Don't have a wrapper fRunUpdater 
+             profiler.stopEvent(eventStart);
```

Comment:
I think it would be better to put this into a finally block at the end of the function to ensure it is called even if there is an error in the framework.